components/card.css

```diff
@@ -0,0 +1,203 @@
+.hz--card-container {
+	display: flex;
+	background-color: var(--teritiary-color);
+	width: 600px;
+	margin: 0 auto;
+	gap: 2rem;
+	border-radius: 5px;
+}
+
+.hz--card-img-box {
+	width: 30%;
+}
+
+.hz--card-img-box img {
+	width: 100%;
+	border-top-right-radius: 10px;
+	border-top-left-radius: 10px;
+}
+
+.hz--card-text-box {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+}
+
+.product-name {
+	font-size: 1.8rem;
+	margin-top: 0.8rem;
+}
+
+.hz--card-btn-box {
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+}
+
+.card-cta-btn {
+	text-decoration: none;
+	border-radius: 5px;
+	text-align: center;
+	font-size: 1.4rem;
+}
+.primary-cta--card {
+	background-color: var(--primary-color);
+	color: var(--secondary-color);
+}
+
+.secondary-cta--card {
+	background-color: var(--secondary-color);
+	color: black;
+}
+
+/***********************/
+/***Vertical card ***/
+/***********************/
+
+.vt--card-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+	align-items: center;
+	background-color: var(--teritiary-color);
+	width: 300px;
+	height: 600px;
```

Comment:
Please give in rem unit, it is good for responsiveness(refactor it later on)

Reply:
fixed